### ISSUE TITLE
Open user manual editing for the crowd

### ISSUE THREAD

**Issue by Lineflyer** [cgeo/cgeo.github.io#226]
Background:
The [c:geo user manual](https://manual.cgeo.org) was created about a year ago to simply have a new user manual for the c:geo app. It currently runs on my webspace with a reverse proxy set up to the above mentioned URL.
I created it myself and over time invited other users to collaborate or translate. Up to now its configured as a closed system, meaning that accounts cannot be self created.

As I learned from recent changes to the c:geo app functionality I cannot manage myself to keep the manual up to date with new features and changed views in c:geo.

So I would suggest to open it up to the public (=our users) to get it hopefully self maintained by the crowd.
Thats actually what a wiki system is designed for.

So basically I would reconfigure it to allow user registration and add authoring rights to all users. Afterwards invite users to participate via our channels.
What still remains for us is to monitor and moderate and maybe to somehow collect the parts of the manual, which need changes due to our changes to the app.

I do not have experience with public wiki pages...anything we should pay attention to?

Should new accounts have the need to be approved by us?

Where could we collect topics, pages, which need to be updated?

**Comment by Lineflyer** [comment]
BTW:
I used the opportunity of my participation on the OC-talk podcast of opencaching.de to promote contributing to our user manual.

**Comment by Lineflyer** [comment]
Duplicate of https://github.com/cgeo/cgeo-guide/issues/5
